### PR TITLE
Remove unnecessary custom_import from train.py

### PR DESCRIPTION
**Pull request by Czm369**
Remove unnecessary custom_import from train.py

### CODE REVIEW

**Comment by ZwwWayne** [comment]
Lint failed, Please click the detail as the picture below, check the log, and fix the lint issues.
image

File: README.md [hunk 1]
```diff
@@ -205,3 +205,4 @@ If you use this toolbox or benchmark in your research, please cite this project.
 - [MMEditing](https://github.com/open-mmlab/mmediting): OpenMMLab image and video editing toolbox.
 - [MMOCR](https://github.com/open-mmlab/mmocr): A Comprehensive Toolbox for Text Detection, Recognition and Understanding.
 - [MMGeneration](https://github.com/open-mmlab/mmgeneration): OpenMMLab image and video generative models toolbox.
+- [MMFlow](https://github.com/open-mmlab/mmflow): OpenMMLab optical flow toolbox and benchmark.
```
Comment (RangiLyu) [inline README.md:208]: Commits that do not belong to this PR are involved.

File: mmdet/apis/train.py [hunk 2]
```diff
@@ -162,7 +162,8 @@ def train_detector(model,
     # register hooks
     runner.register_training_hooks(cfg.lr_config, optimizer_config,
                                    cfg.checkpoint_config, cfg.log_config,
-                                   cfg.get('momentum_config', None))
+                                   cfg.get('momentum_config', None),
+                                   custom_hooks_config=cfg.get('custom_imports', None))
```
Comment (hhaAndroid) [inline mmdet/apis/train.py:166]: custom_imports -> custom_hooks
